modules/tinymce/src/core/main/ts/init/InitContentBody.ts

```diff
@@ -50,8 +51,95 @@ const appendStyle = function (editor: Editor, text: string) {
   Insert.append(head, tag);
 };
 
+const getRootName = (editor: Editor): string => editor.inline ? editor.getElement().nodeName.toLowerCase() : undefined;
+
+const removeUndefined = <T>(obj: T): T => {
+  return Obj.filter(obj as Record<string, unknown>, (v) => Type.isUndefined(v) === false) as T;
+};
+
+const mkParserSettings = (editor: Editor): DomParserSettings => {
+  const settings = editor.settings;
+  const blobCache = editor.editorUpload.blobCache;
+
+  return removeUndefined<DomParserSettings>({
```

Comment:
Hmm, I don't really like that this just asserts the type, as settings could be missing. I'll try and see if theres a way we can do this without having to do the cast today.

Reply:
Ignore sorry, I misread.